salt/metalk8s/saltmaster/minion_configured.sls

```diff
@@ -0,0 +1,31 @@
+{% set saltminion_version = '2018.3.3'%}
+
+
+Configure saltstack repo:
```

Comment:
This step and the next one feel like `minion-installed`, not `minion-configured`, or something.